packages/components/e2e/component-objects/src/main/java/org/talend/component/HeaderBar.java

```diff
@@ -0,0 +1,55 @@
+package org.talend.component;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HeaderBar extends Component {
+
+    private static final Logger LOGGER = LogManager.getLogger(AppHeaderBar.class);
+
+    static final String HEADER_NAME = "AppHeaderBar";
+
+    static final String HEADER_SELECTOR = ".tc-header-bar";
+
+    static final String HELP_SELECTOR = "header-help";
+
+    static final String USER_SELECTOR = "header-user";
+
+    static final String USER_DROP_DOWN_SELECTOR = ".dropdown-menu.dropdown-menu-right";
+
+    static final String ABOUT_SELECTOR = "//li/a[@label='About']";
```

Comment:
is it possible to avoid xpath request ?

Reply:
li > a[label=About]

Reply:
in fact it should be replace by a getBtnById something like that